RFC/src/RFC-0301_NamespaceRegistration.md

```diff
@@ -0,0 +1,236 @@
+
+**Req** - The OpenAlias TXT DNS record MUST be constructed as follows:
+
+| OpenAlias TXT DNS Record Field | OpenAlias TXT DNS Record Data                                |
+| ------------------------------ | ------------------------------------------------------------ |
+| oa1:\<name\>                   | "oa1:tari_raid"                                              |
+| fqdn                           | \<FQDN\>                                                     |
+| public_key                     | \<256 bit public key in hexadecimal format excluding the leading "0x" (64 characters)\> |
+| raid_id                        | \<`RAID_ID` in hexadecimal format excluding the leading "0x" (44 characters)\> |
+| public_nonce                   | \<256 bit public nonce in hexadecimal format excluding the leading "0x" (64 characters)\> |
+| signature                      | \<[Asset issuer]'s 256 bit Schnorr signature (bound to the `RAID_ID`), in hexadecimal format excluding the leading "0x" (64 characters)\> |
+| asset_description              | \<Optional description\>                                     |
+| checksum                       | \<CRC-32 checksum of the entire record up to but excluding the checksum key-value pair, in hexadecimal format excluding the leading "0x" (8 characters)\> |
+
+**Req** - The OpenAlias TXT DNS record MUST adhere to the formatting requirements as specified in [[1]].
+
+
+
+### The RAID_ID
+
+Because the `RAID_ID` does not exist as an entity on the base layer or in the [DAN], it cannot be owned or 
+transferred, but only be verified as part of the OpenAlias TXT DNS record [[1]] verification. If an asset creator 
+chooses not to link a `RAID_ID` and FQDN, a default network assigned `RAID_ID` will be used in the digital asset 
+registration process.
+
+**Req** - A `RAID_ID` MUST be the result of a `Base58Check` encoding of a 256 bit hash function (`Hash256`). This will 
+result in a 44 character string.
+
+**Req** - A default `RAID_ID` MUST be used where it will not be linked to a FQDN, for example it MAY be calculated as 
+`RAID_ID = Base58Check(Hash256("No FQDN"))`.
+
+**Req** - A FQDN linked (non-default) `RAID_ID` MUST be calculated as follows: 
+`RAID_ID = Base58Check(Hash256(PubKey || FQDN))`.
+
+**Req** - A valid `RAID_ID` signature MUST be a 256 bit Schnorr signature defined as `s = PvtNonce + e·PvtKey` with 
+the challenge `e` being `e = Hash256(PubNonce || PubKey || RAID_ID)`.
+
+
+
+### Sequence of Events
+
+The sequence of events leading up to digital asset registration are perceived as follows:
+
+1. The [asset issuer] will decide if the default `RAID_ID` or a `RAID_ID` that is linked to a FQDN must be used for 
+   asset registration.
+
+2. **Req** - If a default `RAID_ID` is required:
+   1. The asset issuer MUST use the default `RAID_ID` (see [The RAID_ID](#the-raid_id)).
+   2. The asset issuer MUST sign the `RAID_ID` as specified (see [The RAID_ID](#the-raid_id)).
```

Comment:
Why do they need to provide a signature when using the default RAID_ID? The signature only lives in the DNS record right? If they are using a Default RAID_ID there is no DNS record?

Reply:
Oh, it is probably being used during the validation by the VNs during the Asset Creation process. This makes sense as the VN's will not have to have a special case for handling the default, they treat them all the same.

Reply:
The thinking here is to force a cognitive choice of RAID_ID and to make the validation code similar.
Makes sense?

Reply:
How is this going to work, are we signing with a known private key or something?

Reply:
We need to weigh a simple if-then in the VN code, i.e.
```
if !new_asset.has_default_raid() { 
  check_raid_signature(...)
}
```

vs. the overhead of creating a new signature for every default-RAID asset.

To me the former is simpler overall.

Reply:
Ok, requirement to sign a default RAID_ID is removed, MUST NOT sign is specified.